polaris-react/src/components/Button/Button.scss

```diff
@@ -383,8 +383,7 @@
   &.iconOnly {
     // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
     @include recolor-icon(var(--p-icon));
-    // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
-    margin: calc(-0.5 * (36px - var(--p-icon-size-medium)));
```

Comment:
these two are equivalent, there would be **no changes visually**. It's nice they tried to tie this value to icon size but the calc is overly complicated and our space tokens should be related to icon size anyways

before:
<img width="233" alt="Screenshot 2022-12-06 at 2 36 59 PM" src="https://user-images.githubusercontent.com/20652326/206011365-b12513e5-0ae5-4386-8583-794d28849d91.png">

after:
<img width="230" alt="Screenshot 2022-12-06 at 2 36 32 PM" src="https://user-images.githubusercontent.com/20652326/206011405-801259e5-77ba-40e4-b526-fd9c967b59f3.png">